keras2onnx/ke2onnx/activation.py

```diff
@@ -34,7 +34,16 @@
                   tf.nn.relu: apply_relu,
                   tf.nn.relu6: apply_relu_6,
                   tf.nn.elu: apply_elu,
-                  tf.nn.tanh: apply_tanh}
+                  tf.nn.selu: apply_selu,
```

Comment:
any idea why the function is not as tf.nn? if it was customized function was created by the user, it will be pretty dangerous to  use the name.

Reply:
Through debugging, I can only see the function has different address from tf.nn.softmax.

Reply:
re-checked, code updated.